src/main/java/net/dv8tion/jda/api/JDA.java

```diff
@@ -1747,6 +1748,62 @@ default List<RichCustomEmoji> getEmojisByName(@Nonnull String name, boolean igno
         return getEmojiCache().getElementsByName(name, ignoreCase);
     }
 
+    /**
+     * Creates a new {@link ApplicationEmoji} for this bot.
+     *
+     * <p>Note that the bot is limited to 2000 Application Emojis (normal and animated).
```

Comment:
This 2000 should be a constant that the user can use, and should be referenced in the docs via the constant

Reply:
Where should I put this constant exactly?

Reply:
I put it into the ApplicationEmoji now.